### PR TITLE
Modify Custom User Store Manager sample

### PR DESCRIPTION
**Pull request by sahandilshan**
## Purpose
* $subject

### CODE REVIEW

File: user-mgt/custom-jdbc-user-store-manager/pom.xml [hunk 1]
```diff
@@ -89,6 +90,11 @@
             <artifactId>org.wso2.carbon.user.core</artifactId>
             <version>${carbon.kernel.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.jasypt</groupId>
+            <artifactId>jasypt</artifactId>
+            <version>1.9.2</version>
```
Comment (piraveena) [inline user-mgt/custom-jdbc-user-store-manager/pom.xml:96]: Add version tag at the end of the pom file. Similar to ` <version>${carbon.kernel.version}</version>`

File: user-mgt/custom-jdbc-user-store-manager/pom.xml [hunk 2]
```diff
@@ -117,10 +123,11 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
                         <Private-Package>
-                            org.wso2.custom.user.store.internal.*
+                            org.wso2.custom.user.store.internal.*,
+                            org.jasypt.*
```
Comment (piraveena) [inline user-mgt/custom-jdbc-user-store-manager/pom.xml:127]: Why this package is added as a private package.

Comment (sahandilshan) [reply user-mgt/custom-jdbc-user-store-manager/pom.xml:127]: The sample application uses the jasypt library to hash the password. So in order to run this, the jasypt library should be available in the runtime. That's why I'm private packaging it.

File: user-mgt/custom-jdbc-user-store-manager/pom.xml [hunk 3]
```diff
@@ -133,6 +140,8 @@
 
                             org.wso2.carbon.user.core.*; version="${carbon.kernel.package.import.version.range}",
                             org.wso2.carbon.user.api.*; version="${carbon.user.api.imp.pkg.version.range}",
+                            org.wso2.carbon.utils.*; version="${carbon.kernel.package.import.version.range}",
+                            *;resolution:=optional
```
[the remark this answers is not in the document]
Comment (sahandilshan) [reply user-mgt/custom-jdbc-user-store-manager/pom.xml:144]: This sample uses the `org.wso2.carbon.utils.Secret` class. That's why I have added it as an import package here.

File: user-mgt/custom-jdbc-user-store-manager/pom.xml (hunk 1 above)
Comment (sahandilshan) [reply user-mgt/custom-jdbc-user-store-manager/pom.xml]: Fixed with https://github.com/wso2/samples-is/pull/330/commits/d9dc0341d2536a042f541d3a1afab4584d3a96eb

**Review by piraveena — APPROVED** [review]
Resolve the merge conflicts in pom file

**Pull request merged**